src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,906 @@
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+  - progressive-web-apps
+---
+
+{% Aside %}
+Web apps should be able to do anything native apps can. The [Capabilities
+project](https://developers.google.com/web/updates/capabilities), of which Web
+NFC is only a part, aims to do just that. To learn about other capabilities and
+to keep up with their progress, follow [Unlocking new capabilities for the
+web](https://developers.google.com/web/updates/capabilities).
+{% endAside %}
+
+## What is Web NFC? {: #what }
+
+Web NFC provides sites the ability to read and write to NFC tags when they are
+brought in close proximity to the user's device (usually 5-10 cm, 2-4 inches).
+The current scope is limited to NDEF, a lightweight binary message format that
+works across different tag formats.
+
+{% Aside 'gotchas' %}
+Web NFC is limited to NDEF because the security properties of reading and
+writing NDEF data are more easily quantifiable. Low-level I/O operations (e.g.
+ISO-DEP, NFC-A/B, NFC-F) and Host-based Card Emulation (HCE) are not supported.
+{% endAside %}
+
+<figure class="w-figure">
+  <img src="./nfc-operation-diagram.png" alt="Phone powering up an NFC tag to exchange data">
+  <figcaption class="w-figcaption">Diagram of an NFC operation</figcaption>
+</figure>
+
+## Suggested use cases {: #use-cases }
+
+Examples of sites that may use Web NFC include:
+- Museums and art galleries can display more information about a display when
+  the user touches their device to a NFC card near the exhibit.
+- Inventory management sites can read or write data to the NFC tag on a
+  container to update information on its contents.
+- Conference sites can use it to scan NFC badges during the event.
+- Sites can use it for sharing initial secrets needed for device or service
+  provisioning scenarios and also to deploy configuration data in operational
+  mode.
+
+<figure class="w-figure">
+  <img src="./nfc-inventory-management.png" alt="Phone scanning several NFC tags">
+  <figcaption class="w-figcaption">NFC inventory management illustrated</figcaption>
+</figure>
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                       | Status                       |
+| ------------------------------------------ | ---------------------------- |
+| 1. Create explainer                        | [Complete][explainer]        |
+| 2. Create initial draft of specification   | [Complete][spec]             |
+| **3. Gather feedback and iterate design**  | [**In Progress**](#feedback) |
+| **4. Origin trial**                        | [**In Progress**][ot]        |
+| 5. Launch                                  | Not Started                  |
+
+</div>
+
+## Using Web NFC {: #use }
+
+### Enabling support during the origin trial phase
+
+Web NFC will be available on Android as an origin trial in Chrome 81. The origin
```

Comment:
link to origin trial?

Reply:
The `{% include 'content/origin-trials.njk' %}` bit below does this.
See https://deploy-preview-2146--web-dev-staging.netlify.com/nfc/#register-for-ot

Reply:
You can get an OT token in Chrome 81. The channel doesn't matter.

Reply:
I was recently told that some origin trials don't work until stable. I wanted to be sure.